pkilint/etsi/en_319_412_5.py

```diff
@@ -307,6 +309,24 @@ def __init__(self):
         )
 
 
+class QcStatementPresenceValidator(extension.ExtensionPresenceValidator):
```

Comment:
I don't think this class is needed, as the `NaturalPersonExtensionIdentifierAllowanceValidator` flags when the QCStatements extension is missing.